### PR TITLE
store AutoScaling/AutoBalancing in Helix

### PR DESCRIPTION
**Pull request by Technoboy-**

### CODE REVIEW

File: uReplicator-Manager/src/main/java/com/uber/stream/kafka/mirrormaker/manager/core/ControllerHelixManager.java [hunk 1]
```diff
@@ -1420,27 +1421,50 @@ private static String getSrc(String pipeline) {
...
   }
 
   public void enableAutoBalancing() {
-    _enableRebalance = true;
+    updateClusterConfig(AUTO_BALANCING, ENABLE);
   }
 
   public boolean isAutoBalancingEnabled() {
-    return _enableRebalance;
+    HelixConfigScope scope = newClusterConfigScope();
```
Comment (yangy0000) [inline uReplicator-Manager/src/main/java/com/uber/stream/kafka/mirrormaker/manager/core/ControllerHelixManager.java]: this code is very similar to `/uReplicator-Controller/src/main/java/com/uber/stream/kafka/mirrormaker/controller/core/HelixMirrorMakerManager.java` 
shall we consider refactor it and put into Utils.

File: uReplicator-Manager/src/main/java/com/uber/stream/kafka/mirrormaker/manager/core/ControllerHelixManager.java [hunk 2]
```diff
@@ -117,14 +122,10 @@
...
   private ZkClient _zkClient;
 
-  private boolean _enableAutoScaling = true;
-  private boolean _enableRebalance;
-
   public ControllerHelixManager(
       KafkaClusterValidationManager kafkaValidationManager,
       ManagerConf managerConf) {
     _conf = managerConf;
-    _enableRebalance = managerConf.getEnableRebalance();
```
Comment (yangy0000) [inline uReplicator-Manager/src/main/java/com/uber/stream/kafka/mirrormaker/manager/core/ControllerHelixManager.java]: shall we keep the `managerConf.getEnableRebalance();` so users can specify the default value during service startup?

Comment (Technoboy-) [reply uReplicator-Manager/src/main/java/com/uber/stream/kafka/mirrormaker/manager/core/ControllerHelixManager.java]: sure

File: uReplicator-Manager/src/main/java/com/uber/stream/kafka/mirrormaker/manager/core/ControllerHelixManager.java (hunk 1 above)
Comment (Technoboy-) [reply uReplicator-Manager/src/main/java/com/uber/stream/kafka/mirrormaker/manager/core/ControllerHelixManager.java]: ok

**Pull request closed without merging**